docs/GITOPS-CONFIG.md

```diff
@@ -20,31 +20,43 @@ If you installed using the [`ai-rhdh-installer`](../README.md#install) all that
 ### AI-RHDH-Installer: Manual Configuration
 
 #### Step 1: ConfigMaps and Secrets
-In your chosen namespace you should apply the following ConfigMaps:
-1.  [`argocd-config.yaml` ](../resources/argocd-config.yaml)
-2.  [`argocd-plugins.yaml`](../resources/argocd-plugins.yaml)
+In your chosen namespace you should apply the `argocd-config` ConfigMap under [`argocd-config.yaml` ](../resources/argocd-config.yaml).
 
 As part of the `ai-rhdh-installer` a secret was created in your desired namespace with the name `rhdh-argocd-secret`, keep note of this Secret as well as the two ConfigMaps applied above.
 
 #### Step 2: Updating Plugins
 **Note: RHDH will encounter errors if the ArgoCD plugins are loaded before an instance is properly attached - This step may result in errors until all steps are completed**
 
-To include the ArgoCD plugins we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
+To include the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) we need to edit the dynamic plugins ConfigMap that was created by the RHDH Operator:
 
 ![Dynamic Plugins Example](../assets/dynamic-plugins-example.png)
 
-Edit the associated `yaml` file to include `argocd-plugins.yaml` under the `includes` section:
+Edit the associated `yaml` file to include the contents of the [ArgoCD plugins list](../dynamic-plugins/argocd-plugins.yaml) under the `plugins` section:
 
 ![Dynamic Plugins Example 2](../assets/dynamic-plugins-example-2.png)
 
-#### Step 3: Updating RHDH Deployment
-Now that all of the required ConfigMaps and Secrets are apart of the namespace, as well as adding the plugins we must edit the RHDH Deployment to properly reference these items.
-
-Since we are referencing `argocd-plugins.yaml` in our dynamic-plugins ConfigMap we first must add `argocd-plugins.yaml` as a `volumeMount` to the `initContainers` field in the RHDH Deployment under `.spec.template.spec.initContainers.volumeMounts`
+Alternatively, we can use this series of commands to perform the same task with `kubectl` and `yq` using the [`argocd-plugins.yaml`](../dynamic-plugins/argocd-plugins.yaml):
```

Comment:
Might be more clear here if we have 2 sub-sections:

* Edit the associated `yaml` file to include..
* OR Alternatively, we can use this series...

Reply:
Also done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/143b26952fac4d4be9bb4d0942fab099b69d7dc5

Reply:
Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/5659a91ba4bc3ba06bb2b68130e890d0d1fa24e9